### ISSUE TITLE
Accessible section about screenreader friendly prompting is not accessible

### ISSUE THREAD

**Issue by williammartin** [charmbracelet/huh#611]
### Describe the bug

The [accessibility section](https://github.com/charmbracelet/huh?tab=readme-ov-file#accessibility) has a moving image showing how the prompts appear, but someone using a screenreader would probably have no idea what is happening.

Perhaps we could include textual examples of each form type in code blocks?

**Comment by caarlos0** [comment]
this is a good call!

I'm working on improve the accessible prompts as well, see #620 and #623 if you wanna take a look.
